src/util/linear_scan.rs

```diff
@@ -75,7 +76,7 @@ impl<VM: VMBinding> LinearScanObjectSize for DefaultObjectSize<VM> {
 
 /// Region represents a memory region with a properly aligned address as its start and a fixed size for the region.
 /// Region provides a set of utility methods, along with a RegionIterator that linearly scans at the step of a region.
-pub trait Region: Copy + PartialEq + PartialOrd {
+pub trait Region: Copy + PartialEq + PartialOrd + Debug {
```

Comment:
Is `Debug` really needed? I am okay with this if it is intended. I was just reminded of this, as it changed our public API.

Reply:
fixed